packages/legend-extension-external-language-morphir/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@finos/legend-extension-external-language-morphir",
+  "version": "0.0.1",
+  "description": "Legend extension for Morphir external language",
+  "keywords": [
+    "legend",
+    "legend-extension",
+    "external-language",
+    "external-language-morphir",
+    "morphir"
+  ],
+  "homepage": "https://github.com/finos/legend-studio/tree/master/packages/legend-extension-external-language-morphir",
+  "bugs": {
+    "url": "https://github.com/finos/legend-studio/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/finos/legend-studio.git",
+    "directory": "packages/legend-extension-external-language-morphir"
+  },
+  "license": "Apache-2.0",
+  "sideEffects": false,
+  "type": "module",
+  "main": "lib/index.js",
+  "module": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "build": "yarn clean && yarn build:tsc",
+    "build:tsc": "tsc --project ./tsconfig.build.json",
```

Comment:
I would expect some handling of styles here as well, you would have some styles pertained to this package. Again, please refer to the template from `dsl-data-space` for example.

Reply:
Didn't change since eventually didn't use a local stylesheet for the extension.